tests/test_data_processing.py

```
@@ -188,246 +186,258 @@ def mock_tokenized_inputs(self):
         types = ["A"]
         fwd, rev = make_mapping(types)
-        
+
         assert fwd["A"] == 1
         assert rev[1] == "A"
-    
+
     def test_deduplicates_while_preserving_order(self):
         """Should remove duplicates while keeping first occurrence order."""
         types = ["PER", "LOC", "PER", "ORG", "LOC"]
         fwd, rev = make_mapping(types)
```

Ruff (0.15.1)

[warning] Unpacked variable `rev` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)